mlcomp/tests/test_mlpiper_cmdline.py

```diff
@@ -147,14 +147,17 @@ def setup_class(cls):
         # os.chdir(mlcomp_root_path)
         # subprocess.check_call("make egg", shell=True)
 
-        mlcomp_eggs = glob.glob(os.path.join(mlcomp_root_path, "dist", "*.egg"))
+
+        eggs_pattern = "*2.*.egg" if sys.version_info[0] < 3 else "*3.*.egg"
```

Comment:
I would use `py2`/`py3` instead of `2`/`3` correspondingly

Reply:
fixed